webcomponents/tool/src/main/frontend/js/grader/sakai-grader.js

```diff
@@ -694,20 +703,30 @@ export class SakaiGrader extends gradableDataMixin(SakaiElement) {
     this.submittedTextMode = true;
   }
 
+  selectAttachment(e) {
```

Comment:
this shouldn't take the event as an argument. It would be better to pass the actual url from the event in the caller.
selectAttachment(e.target.dataset.url);